.github/actions/build-setup/action.yml

```diff
@@ -55,7 +81,7 @@ runs:
         echo 'DIEM_DUMP_LOGS=1' | tee -a $GITHUB_ENV
         echo 'CARGO_INCREMENTAL=0' | tee -a $GITHUB_ENV
         echo 'RUST_NIGHTLY='$(cat cargo-toolchain) | tee -a $GITHUB_ENV
-        echo 'CARGO_HOME=/opt/cargo/' | tee -a $GITHUB_ENV
```

Comment:
yes, to make it available to the devsetup.sh script in macos, in linux it's the base image already has all the tooling installed and this would all be about upgrading.   And the docker image relies on symlinks which behave differently in mac osx unfortuntaly.